core/templates/dev/head/pages/exploration_player/TutorCardDirective.js

```diff
@@ -137,7 +133,7 @@ oppia.directive('tutorCard', [
           $scope.profilePicture = UrlInterpolationService.getStaticImageUrl(
             '/avatar/user_blue_72px.png');
 
-          ExplorationPlayerService.getUserProfileImageAsync()
+          ExplorationEngineService.getUserProfileImageAsync()
```

Comment:
This is absolutely not the job of ExplorationEngineService :)

OK to make separate services for the other functions, or keep ExplorationPlayerService, or something. The focus of this PR should be to get a clean definition of ExplorationEngineService (and perhaps the Card object), so that we can do something similar for pretests. We can do the rest of the refactor later, in stages.

Reply:
Done.